Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -742,6 +750,21 @@ def run(self):
             self.command += ['--test-writer=ruby']
 
         self.appendCustomBuildFlags(platform, self.getProperty('fullPlatform'))
+        self.command = ['/bin/sh', '-c', ' '.join(self.command) + ' 2>&1 | python3 Tools/Scripts/filter-jsc-tests.py']
```

Comment:
Does it work fine with WithProperties?

Reply:
Yeah, `self.command` is handled by buildbot